components/Steps/index.jsx

```diff
@@ -230,15 +230,17 @@ export const inputLabels = {
       validation: {
         required: true,
         pattern: {
-          value: /^[0-9]{8}$/,
+          value: /^\d{8}$/,
         },
       },
     },
     businessIdentifyNumberVAT: {
+      type: 'number',
+      hint: 'Please enter your 9 digits number',
       validation: {
         required: true,
         pattern: {
-          value: /^[0-9a-z]{11}$/i,
+          value: /^\d{9}$/i,
```

Comment:
We can remove the case insensitive validator from the RegEx validator.

```suggestion
          value: /^\d{9}$/,
```